### ISSUE TITLE
Properties are not properly inherited between a ActiveRel and its subclasses

### ISSUE THREAD

**Issue by nieluj** [neo4jrb/neo4j#1080]
The code below exhibits the faulty behaviour:

```ruby
require 'neo4j'

Neo4j::Session.open(:server_db, "http://localhost:7474", { basic_auth: { username: 'neo4j', password: 'password'} })

class Post
  include Neo4j::ActiveNode
end

class Comment
  include Neo4j::ActiveNode
end

class GenericRel
  include Neo4j::ActiveRel
  from_class :any
  to_class :any

  property :confidence, type: String, default: "Unknown"
end

class SpecificRel < GenericRel
  from_class Post
  to_class Comment
end

p = Post.create
c = Comment.create

r = SpecificRel.new
r.from_node = p
r.to_node = c
r.confidence = "High"
r.save
```

Running this code raises an exception:

```console
/home/jipe/.rvm/gems/ruby-2.2.1/gems/active_attr-0.8.5/lib/active_attr/typecasted_attributes.rb:99:in `_attribute_type': undefined method `[]' for nil:NilClass (NoMethodError)
        from /home/jipe/.rvm/gems/ruby-2.2.1/gems/active_attr-0.8.5/lib/active_attr/typecasted_attributes.rb:67:in `_attribute_type'
        from /home/jipe/.rvm/gems/ruby-2.2.1/gems/active_attr-0.8.5/lib/active_attr/typecasted_attributes.rb:75:in `_attribute_typecaster'
        from /home/jipe/.rvm/gems/ruby-2.2.1/gems/neo4j-6.0.3/lib/neo4j/shared/property.rb:201:in `block in attribute!'
        from fu.rb:32:in `<main>'
```

**Comment by cheerfulstoic** [comment]
Thanks for reporting this!

@subvertallchris I looked at this some, but I'm not quite sure what's going on.  It looks like the @attributes class var from ActiveAttr isn't being inherited

**Comment by subvertallchris** [comment]
This fix was released as part of 6.1. Thanks for reporting!
